index.html

```diff
@@ -53,5 +54,237 @@ <h1 >
         </div>
     </section>
 
+    <section id="work-section">
+        <h1>
+            <span id="work-heading">
+                My Recent<br>
+                Work
+            </span>
+        </h1>
```

Comment:
- [ ] Kindly I recommend  you to uses a header tag` hx(h2)` for the title (e.g., "My Recent Works") instead of `h1` tag for the purpose to keep HTML Hierarchy and search and optimization.

Reply:
Thank you so much for highlighting all these issues I am fixing all of these for the new pull request